### PR TITLE
Premake scripts fixed

### PR DESCRIPTION
**Pull request by nezvers**
* root script had a typo for TPOrbitCameraCSample folder. No more missing raylib.h.
* Premake/Game script had wrong file/ directories
* Premake/raylib scripts had wrong file/ directories
- Root premake creates a solution but has CloseWindow already defined ERROR.

### CODE REVIEW

File: Premake/Game/premake5.lua [hunk 1]
```diff
@@ -47,13 +47,13 @@ project "raylib"
...
+		includedirs { "../../raylib/src", "../../raylib/src/external/glfw/include"}
 		vpaths 
 		{
-			["Header Files"] = { "raylib/src/**.h"},
-			["Source Files/*"] = {"raylib/src/**.c"},
+			["Header Files"] = { "../../raylib/src/**.h"},
+			["Source Files/*"] = {"../../raylib/src/**.c"},
 		}
-		files {"raylib/src/*.h", "raylib/src/*.c"}
+		files {"../../raylib/src/*.h", "../../raylib/src/*.c"}
```
Comment (JeffM2501) [inline Premake/Game/premake5.lua:57]: These premake scripts are not intended to be run from here, but the basis for a new project. They will be copied to a new root folder. Thus these folder changes will break that process. See this tutorial for how they are intended to be used
https://github.com/raysan5/raylib/wiki/Easy-Raylib-Setup-for-Windows-with-Visual-Studio

File: premake5.lua [hunk 2]
```diff
@@ -74,10 +74,10 @@ project "rlExtrasC"
 	includedirs { "raylib/src","rlExtrasC"}
 	vpaths 
 	{
-		["Header Files"] = { "rlExtrasC/**.h"},
+		["Header Files"] = { "rlExtrasC/**.h", "raylib/src/raylib.h"},
 		["Source Files"] = {"rlExtrasC/**.c"},
 	}
-	files {"rlExtrasC/**.h", "rlExtrasC/**.c"}
+	files {"rlExtrasC/**.h", "rlExtrasC/**.c", "raylib/src/raylib.h"}
```
Comment (JeffM2501) [inline premake5.lua:80]: These are to add the header file to the project for editing. They are not needed for include dirs. I would prefer to not directly add raylib.h to the project and just let the include dirs handle finding it. More than just raylib.h is used.

File: premake5.lua [hunk 3]
```diff
@@ -89,16 +89,17 @@ project "rlExtrasCPP"
 	filter{}
 	
 	location "./rlExtrasCPP/"
-	language "C"
+	language "C++"
+	cppdialect "C++17"
```
Comment (JeffM2501) [inline premake5.lua:93]: What specifically needs C++17? I'd like to keep the smallest C++ required.

**Review by JeffM2501 — CHANGES_REQUESTED** [review]
Thank you for the review and changes, but I think there needs to be some changes before this can be accepted (see notes).

File: premake5.lua (hunk 3 above)
Comment (nezvers) [reply premake5.lua:93]: My bad. But shouldn't `rlExtrasCPP` be C++?

**Pull request closed without merging**